.github/workflows/license.yml

```diff
@@ -0,0 +1,27 @@
+name: Check license
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
does it mean pull request against `master`? traditionally we didnt have a branch for `pull_request` events.

Reply:
Oh my bad. I'll update